src/Engine/ProtoCore/Lang/FunctionEndPoint.cs

```diff
@@ -25,16 +25,6 @@ public abstract class FunctionEndPoint
 			set;
 		}
 		
-		public bool FuncHasOverloads { //If the function has overloads and the type distance has changed, we'll have to do a re-resolution
```

Comment:
Except for mirror and AST APIs, VM APIs are not to be used by users. Hence it's ok to break this one.